docs/release_notes/plugin_api/0.1.x.rst

```diff
@@ -0,0 +1,8 @@
+Plugin API 0.1.alpha Release Notes
```

Comment:
I'm not entirely sure if we're doing a x.y.z or x.y version scheme for the plugin api. If it's x.y then this is right but the first should be called `0.1.rst` if it's x.y.z this should probably be 0.0.1.alpha instead. What do you think?